www/scipylib/building/windows.rst

```diff
@@ -32,6 +32,109 @@ information than this document will provide.
 .. _OpenBLAS: https://github.com/xianyi/OpenBLAS
 .. _LAPACK: http://www.netlib.org/lapack/
 
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
+we're mostly just going to breeze through this. More in-depth documentation is provided 
+below.
+
+First, download all of the following software concurrently:
+
+1) MinGW-w64 from https://mingw-w64.org
+2) Microsoft Visual Studio 2015 or 2017 Community Edition (available from Microsoft)
+3) git from https://git-scm.org/
+4) MSYS2 from http://www.msys2.org/
+5) Python **with pip** from https://python.org/
```

Comment:
They might get confused about "with pip" here - it gets installed by default.  Maybe "; if you customize the install, make sure you leave the option to install "pip" checked".